### ISSUE TITLE
[Linux] [CentOS] [2.2.0] owncloud client fails to present a GUI interface

### ISSUE THREAD

**Issue by JulianTosh** [owncloud/client#5084]
### Expected behaviour
Expect to see a GUI that prompts for connection information

### Actual behaviour
process appears to hang. ~/.local/data/share/data/owncloud folder gets created. Log file displays information that appears to be related to a misconfiguration or missing package related to qt. 

### Steps to reproduce
1. Install owncloud-client on Fedora 23 'dnf install owncloud-client -y'
2. Run 'owncloud'
3. 

### Server configuration
Operating system: CentOS 7.2.1511

Web server: httpd 2.4.6

Database: mysql

PHP version: 5.4.16

ownCloud version: 9.1.0-1.1

Storage backend:

### Client configuration
Client version: 2.2.0-4
Operating system: Fedora 23
OS language: English

Installation path of client:
/usr/bin/owncloud

### Logs

Attached

**Comment by guruz** [comment]
2.2.0 is old.
Can you please try the packages from our website, 2.2.3 beta1?
https://owncloud.org/install/#testing-development

**Comment by JulianTosh** [comment]
I was able to upgrade easily by installing the repo at 
```
https://software.opensuse.org/download/package?project=isv:ownCloud:desktop&package=owncloud-client
```
Only slight problem was 'dnf install owncloud-client' was not sufficient to get the correct version installed. I had to specify the exact version number available in the isv:ownCloud repository, otherwise, it kept installing the old version from the Fedora repository.

This solved my problem. Client works great now.

Thank you so much.
